plugins/modules/swim_workflow_manager.py

```diff
@@ -1799,8 +1813,10 @@ def get_diff_activation(self):
         self.complete_successful_activation = False
         self.partial_successful_activation = False
         self.single_device_activation = False
+        device_ip = self.want.get("activation_details").get("device_ip_address")
 
         if self.have.get("activation_device_id"):
```

Comment:
Same as like getting distribution_device_id...

Reply:
updated the code